package.json

```
@@ -5,8 +5,10 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "watch": "ng build --watch --configuration development",
-    "test": "ng test",
+    "watch": "ng build --watch --configuration production",
```

```suggestion
    "build:prod": "ng build --watch --configuration production",
```
build product thì nên đặt tên như thế

---

@tritdm121099 dạ em fix rồi ạ